src/devices/Lm75/Lm75.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Lm75
+{
+    /// <summary>
+    /// Digital Temperature Sensor LM75
+    /// </summary>
+    public class Lm75 : IDisposable
+    {
+        I2cDevice _sensor;
+
+        /// <summary>
+        /// LM75 I2C Address
+        /// </summary>
+        public const byte I2cAddress = 0x48;
+
+        #region prop
+
+        /// <summary>
+        /// LM75 Temperature (℃)
+        /// </summary>
+        public double Temperature { get => GetTemperature(); }
+
+        private bool _shutdown;
+        /// <summary>
+        /// LM75 Shutdown
+        /// </summary>
+        public bool Shutdown { get { return _shutdown; } set { SetShutdown(value); _shutdown = value; } }
```

Comment:
I would go with some word with past tense i.e. Disabled. Shutdown sounds like a function name

Reply:
Aha, the datasheet says shutdown. I will change the name.😄